src/Entity/Event.php

```diff
@@ -578,4 +578,79 @@ public function isMixDeskValid()
             return true;
         }
     }
+
+    /**
+     * @return string
+     */
+    public function createGoogleLocation()
+    {
+        if ($this->location) {
+            return sprintf('%s, %s', $this->getLocation(), $this->getRegion()->getName());
+        }
+        else {
+            return $this->getRegion()->getName();
+        }
+    }
+
+    /**
+     * @return string
+     */
+    public function createGoogleDescription()
+    {
+        $googleDescription =
```

Comment:
@daria-sieroshtan 
This function is so complex that you may want to use Twig to render this